dpnp/dpnp_iface_statistics.py

```diff
@@ -506,7 +550,30 @@ def _run_native_sliding_dot_product1d(a, v, l_pad, r_pad):
     return out
 
 
-def correlate(a, v, mode="valid"):
+def _convolve_fft(a, v, l_pad, r_pad, rtype):
+    assert a.size >= v.size
+    assert l_pad < v.size
+
+    # +1 is needed to avoid circular convolution
+    padded_size = a.size + r_pad + 1
+    fft_size = 2 ** math.ceil(math.log2(padded_size))
```

Comment:
Since `math.ceil` returns `float`, it should be converted to integer:

```suggestion
    fft_size = 2 ** int(math.ceil(math.log2(padded_size)))
```